crossdc-producer/src/main/java/org/apache/solr/update/processor/ProducerMirroringMetrics.java

```diff
@@ -0,0 +1,43 @@
+package org.apache.solr.update.processor;
+
+import com.codahale.metrics.Counter;
+import com.codahale.metrics.Histogram;
+import org.apache.solr.core.SolrCore;
+import org.apache.solr.metrics.SolrMetricsContext;
+
+public class ProducerMirroringMetrics {
+
+    private final Counter savedDocuments;
+    private final Counter messages;
+    private final Counter tooLargeDocuments;
+    private final Counter mirrorFailures;
+    private final Histogram documentsSize;
+
+    public ProducerMirroringMetrics(SolrMetricsContext solrMetricsContext, SolrCore solrCore) {
```

Comment:
Please add the class-level and property-level javadocs. Some of the metric names are unclear or redundant (eg. "errors.mirroredFailures").

Reply:
Added javadocs.